src/tests/ilp-kit-base.js

```diff
@@ -0,0 +1,333 @@
+/*global describe, it, beforeEach, before, after*/
+'use strict'
+
+const path = require('path')
+const ServiceManager = require('five-bells-service-manager')
+const KitManager = require('../lib/kit-manager')
+const request = require('superagent')
+const assert = require('assert')
+const spawn = require('co-child-process')
+
+const services = new ServiceManager(
+  path.resolve(process.cwd(), 'node_modules/'),
+  path.resolve(process.cwd(), 'data/'))
+const kitManager = new KitManager(services)
+
+// sleep time expects milliseconds
+function sleep (time) {
+  return new Promise((resolve) => setTimeout(resolve, time))
+}
+
+function assertStatusCode (resp, expectedStatus) {
+  assert.equal(resp.statusCode, expectedStatus, `HTTP status code is ${resp.statusCode},
+        but expected is ${expectedStatus}`)
+}
+
+// sets up Apache as a reverse-proxy and for handling SSL connections
+function * setupApache () {
+  const image = process.env.CIRCLE_BUILD_IMAGE
+  if (!image) {
+    console.log('WARN: Not running on CircleCI. ' +
```

Comment:
ilp-kit-base.js is what used to be ilp-kit.js - I'll change this to console.warn, but it's not part o the new code I introduced.